db/table_cache.cc

```diff
@@ -317,7 +328,7 @@ Status TableCache::GetRangeTombstoneIterator(
   TypedHandle* handle = nullptr;
   if (t == nullptr) {
     s = FindTable(options, file_options_, internal_comparator, file_meta,
-                  &handle);
+                  &handle, TableReaderCaller::kUncategorized);
```

Comment:
There's a surprising amount of `kUncategorized`. Are our other favorite read settings (such as IOPriority) plumbed to here through `ReadOptions`?

Reply:
Seems no. Still, I wonder if we should tie completeness of IOActivity to completeness of ReadOptions by making IOActivity a ReadOptions member. Then there's only one thing we have to complete plumbing for. Speaking of that, I did more plumbing of ReadOptions in #11208, which should at least ensure tail reads during `NewTableReader()` are covered

Reply:
Fixed